packages/embeds/embed-core/src/embed.ts

```diff
@@ -36,13 +42,13 @@ customElements.define("cal-inline", Inline);
 
 declare module "*.css";
 type Namespace = string;
-type InitConfig = {
```

Comment:
Renamed InitConfig -> CalConfig as "init" is just how it is set. This was ambiguous with initialConfig